.github/tests/node-selectors.yaml

```diff
@@ -0,0 +1,13 @@
+participants:
+  - el_client_type: reth
+    cl_client_type: teku
+    cl_split_mode_enabled: true
+    node_selectors: {
+      "kubernetes.io/hostname": testing-1,
+    }
+  - el_client_type: reth
+    cl_client_type: teku
+    cl_split_mode_enabled: true
+global_node_selectors: {
+  " kubernetes.io/hostname": testing-2,
```

Comment:
extra space

Reply:
seems like this test isn't running? otherwise why isn't it failing

Reply:
this should be moved into Circle and run as a k8s test!

Reply:
If we move it to Circleci, then its going to continuously fail. I rather leave it as is, and let it not fail.